packages/core/src/sass/account-wizard.scss

```diff
@@ -445,3 +474,216 @@
+        height: 4rem;
+    }
+
+    &__side-note {
+        text-align: justify;
+        padding: 0 7rem;
+
+        @include mobile() {
+            padding: 0 1.6rem;
+        }
+    }
+
+    &__header {
+        margin-top: 2.4rem;
+        padding: 0 7rem;
+
+        @include mobile() {
+            padding: 0 1.6rem;
+        }
+        position: relative;
+        &--background {
+            background-color: var(--general-active);
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+        }
+
+        &--arrow-down {
+            width: 0;
+            height: 0;
+            border-left: 8px solid transparent;
+            border-right: 8px solid transparent;
+            border-top: 16px solid var(--general-active);
+            position: relative;
+            left: 49%;
+        }
+
+        &--content_position {
+            justify-content: center !important;
+        }
+    }
+
+    &__form {
+        margin-top: 2.2rem;
+        display: grid;
+        grid-template-rows: 4.7fr 1fr;
+
+        &--existing_user {
+            grid-template-rows: 5.8fr 1fr;
+        }
+
+        &--layout {
+            grid-template-rows: 35rem 1fr !important;
+        }
+
+        &--fields {
+            display: flex;
+            justify-content: center;
+            padding: 0 7rem;
+            @include mobile() {
+                padding: 0 1.6rem;
+            }
+        }
+
+        .dc-themed-scrollbars {
+            width: 100%;
+        }
+
+        .dc-dropdown__display .dc-dropdown__display-placeholder-text-limit-fit {
+            max-width: unset;
+            padding-bottom: 0.5rem;
+            @include mobile() {
+                width: 90%;
+                white-space: break-spaces;
+            }
+        }
+    }
+
+    &__wrapper__question {
+        .dc-radio-group {
+            gap: 1.6rem;
+        }
+        @include mobile() {
+            .dc-radio-group__circle {
+                align-self: flex-start !important;
+            }
+        }
+    }
+
+    &__wrapper__dropdown {
+        gap: 2.4rem;
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+    }
+
+    .disable-pointer {
+        cursor: not-allowed;
+    }
+
+    .highlight {
+        animation: bounce 2s ease infinite;
+    }
+    @keyframes bounce {
+        0% {
+            transform: scale(1);
+        }
+        50% {
+            transform: scale(2);
+        }
+        100% {
+            transform: scale(1);
+        }
+    }
+
+    @include mobile() {
+        .dc-dropdown__display-placeholder {
+            width: 100%;
+            .trading-assessment__form .dc-dropdown__display .dc-dropdown__display-placeholder-text-limit-fit {
+                width: 90%;
+            }
+        }
+
+        .dc-dropdown__display-placeholder--is-left-text.dc-dropdown__display-placeholder--is-title {
+            white-space: break-spaces !important;
```

Comment:
What is `break-spaces`? 👀

Reply:
It is same as `white-space:normal`. For wrapping training space